mlir/include/mlir/Dialect/LLVMIR/LLVMAttrDefs.td

```diff
@@ -261,15 +262,35 @@ def LLVM_DITagParameter : LLVM_DIParameter<
   "tag", /*default=*/"", "Tag"
 >;
 
+def LLVM_DIExpressionOpcodeParameter : LLVM_DIParameter<
```

Comment:
Just a suggestion:

What do you think about naming it LLVM_DIOperationEncodingParameter to follow the LLVM naming?